### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.8 es 4.38 (6 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#314] [Esercizi M-Z]
![es.38](https://github.com/user-attachments/assets/8adcdd45-d08d-44e5-9e9e-5b1767e1be78)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/6d3e4cb2-1954-4909-976f-73c03e9555cb)
